lib/utils/transform.py

```diff
@@ -6,7 +6,7 @@
 
 # Note: 'sequence' should NEVER appear in these lists!
 METADATA_COLUMNS = [  # Ordering of columns in the existing metadata.tsv in the ncov repo
-    'strain', 'virus', 'gisaid_epi_isl', 'genbank_accession', 'genbank_accession_rev', 'sra_accession', 'date', 'region',
+    'strain', 'virus', 'gisaid_epi_isl', 'genbank_accession', 'genbank_accession_rev', 'sra_accession', 'internal_id', 'date', 'region',
```

Comment:
We are using the `restval` parameter for `csv.DictWriter`, so all columns are included in the output:

https://github.com/nextstrain/ncov-ingest/blob/36eaea56c00876854802bd04ef4db2e944832576/bin/transform-gisaid#L228-L235

It will just be filled with "?" in the column.

Reply:
Hmm, I just ran with the `debug_sample_gisaid.yaml` config and it didn't show up. If it's not in the dict `metadata_fh` it's maybe not output at all.

Reply:
Fixed in https://github.com/nextstrain/ncov-ingest/pull/365/commits/265e961e2cc1a8b06c99b33d9d726d0429d45a17